### PR TITLE
New theme

### PR DESCRIPTION
**Pull request by PierreRambaud**
On desktop with errors and sucess.
image
image

On mobile / tablet:

After with errors:
image

With sucess:
image

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
So beautiful, we have our new front end developer !!

**Comment by PierreRambaud** [comment]
@mickaelandrieu https://vuetifyjs.com/en/  https://nuxtjs.org/ I did nothing, nice try :smiley:

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+RUN mkdir -p /app
+EXPOSE 3000
+
+COPY . /app
+WORKDIR /app
+RUN npm install
+RUN npm run build
+
+ENV HOST 0.0.0.0
```
Comment (Quetzacoalt91) [inline Dockerfile]: What is this line for?

File: pages/index.vue [hunk 2]
```diff
@@ -153,12 +253,102 @@
...
+        background-color: $primary;
+      }
+    }
+    tbody {
+      tr {
+        &:not(:last-child) {
+          border: none;
+        }
+        &.error {
+          background-color: rgba(244, 67, 54, .1) !important;
```
Comment (Quetzacoalt91) [inline pages/index.vue]: I guess this should work even without the `!important` mark. Please check and remove it if possible. Also true for the other occurrences.

Comment (PierreRambaud) [reply pages/index.vue]: No it won't work without the !important because of material js :/

File: Dockerfile (hunk 1 above)
Comment (PierreRambaud) [reply Dockerfile]: Binding to 0.0.0.0 means accepting connections from anywhere.

File: pages/index.vue [hunk 3]
```diff
@@ -1,112 +1,181 @@
...
-              variant="success"
-              @click.stop="details(cell.item, cell.index)"
-              v-if="cell.item.stats"
-            >
-              View Report
-            </b-btn>
-          </template>
-        </b-table>
-      </div>
+            Sorry, nothing to display here :(
```
Comment (matks) [inline pages/index.vue]: 😄

**Pull request merged**